packages/flame/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## [next]
+ - Added possibility to extend `JoystickComponent` (made `knob` unrequired)
```

Comment:
This should be in the bottom of the list

Reply:
Updated